docs/content/manual/release-specific/v1.1.0/uninstallation.md

```diff
@@ -61,3 +62,49 @@ title: Test uninstallation
 
 3. Start to uninstall longhorn.
 4. Check if the uninstaller pod gets stuck there
+
+## Performance improvement of uninstallation
+1. Install Longhorn v1.1.0.
+2. Uninstall longhorn v1.1.0.
+3. Uninstallation should be about 60 seconds faster in compare to Longhorn v1.0.2.
+4. There should not be repeated log messages like below:
```

Comment:
We can add an estimated time range for the uninstallation.

Reply:
Added. Thanks